src/app/orgs/[orgId]/page.tsx

```diff
@@ -1,46 +1,66 @@
-/*
-  These route segment configs should bust Data and Route caching
-  So that the list of users is always up-to-date.
-  More info: https://nextjs.org/docs/app/building-your-application/caching#segment-config-options
-*/
-export const dynamic = 'force-dynamic';
-export const revalidate = 0;
+import Image from 'next/image';
```

Comment:
We need to use our custom `Image` component in order to remove the `style="color:transparent"` the Next.js component adds to image.